corehq/apps/hqwebapp/views.py

```diff
@@ -520,7 +521,7 @@ def assert_initial_page_data(request):
 @require_superuser
 def debug_notify(request):
     try:
-        0 / 0
+        0 // 0
```

Comment:
This is 🍌 🍌 
Looks old, I wonder if at the time `notify_exception` required a real exception?  Could probably just drop all this.

Reply:
I admit I just thought "integer division, great" and stopped thinking about it... @dannyroberts do you know offhand if we can drop this? I don't see any references to it.